ext/mbstring/tests/mb_http_input_multi_post.phpt

```diff
@@ -0,0 +1,44 @@
+--TEST--
+mb_http_input() with POST method and multiple candidate encodings
+--EXTENSIONS--
+mbstring
+--POST--
+a=日本語&b=ελληνικά
+--INI--
+mbstring.encoding_translation=1
+mbstring.http_input=UTF-8,SJIS,EUC-JP,ISO-8859-1
+--FILE--
+<?php
+
+echo $_POST['a']."\n";
+echo $_POST['b']."\n";
+
+// Get encoding
+var_dump(mb_http_input('P'));
+var_dump(mb_http_input('G'));
+var_dump(mb_http_input('C'));
+var_dump(mb_http_input('S'));
+var_dump(mb_http_input('I'));
+var_dump(mb_http_input('L'));
+
+?>
+--EXPECT--
+日本語
+ελληνικά
+string(5) "UTF-8"
+bool(false)
+bool(false)
+bool(false)
+array(4) {
+  [0]=>
+  string(5) "UTF-8"
+  [1]=>
+  string(4) "SJIS"
+  [2]=>
+  string(6) "EUC-JP"
+  [3]=>
+  string(10) "ISO-8859-1"
+}
+string(28) "UTF-8,SJIS,EUC-JP,ISO-8859-1"
+
+Deprecated: PHP Startup: Use of mbstring.http_input is deprecated in Unknown on line 0
```

Comment:
Have you tried to check if this behaves as expected if only [input-encoding](https://www.php.net/manual/en/ini.core.php#ini.input-encoding) is set?

Reply:
Yes, that seems to work! Thanks.